### PR TITLE
Fix text and image spacing

### LINKED ISSUE #207: our mission fixes
image

CURRENT

fix the spacing so it's more 1 to 1 w the figma and round corners for the image 

WHAT WE WANT (corners are not rounded but we want them rounded for the image)

image

spacing is also cooked on mobile so fix that

image

should be even amount of space above the paragraph and below the paragraph to the image

### PR DESCRIPTION
**Pull request by duvalincus**
image
iPhone SE

image
laptop

### CODE REVIEW

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
you have mobile exactly how the witl president wants but for desktop/monitors, we want to decrease the spacing on the left/right of the image. you can test grid-cols-3 and making the text span two columns so there's less space for the image 

below is an image of the dev branch and you can see the gap between the left/right of the image

image

sry that the figma doesn't accurately represent what I'm tryna say rn, ask if you need clarification.

tldr, keep mobile the same as you have it rn but modify desktop version (on dev) to have less spacing on left/right of img

**Comment by duvalincus** [comment]
image
Tried col-span-2 and decreased padding on the left.
